app/controllers/dashboards_controller.rb

```diff
@@ -16,12 +16,17 @@ def show
     if params[:which] == "organization" && params[:org_id] && (@user.org_admin?(params[:org_id]) || @user.any_admin?)
       target = @organizations.find_by(id: params[:org_id])
       @organization = target
+      @articles = target.articles
+    else
+      # if the target is a user, we need to eager load the organization
+      @articles = target.articles.includes(:organization)
```

Comment:
here we were eager loading the organization even when the target was an organization, so I removed that